### DISCUSSION TITLE
Onedrive client for Linux blocked

### DISCUSSION THREAD

**Discussion by ilyacarey** [abraunegg/onedrive#2417] [General]
Hello,
I'm running Ubuntu 22.04.2 on my work (university) computer and I want to back up files with my work Onedrive account. 
However, I get this error when trying to get the response uri:

"AADSTS50105: Your administrator has configured the application OneDrive Client for Linux ('d50ca740-c83f-4d1b-b616-12c519384f0c') to block users unless they are specifically granted ('assigned') access to the application. The signed in user '*******@*** is blocked because they are not a direct member of a group with access, nor had access directly assigned by an administrator. Please contact your administrator to assign access to this application."

I've requested access from the IT department but they said they don't support for Onedrive client for Linux.
Is there any way to get around this?

Thank you.

**Comment by abraunegg** [comment]
@ilyacarey 
> I've requested access from the IT department but they said they don't support for Onedrive client for Linux.

They do not need to support it - they just need to approve it's access. If they are open to having a pragmatic discussion with me so that they can gain further clarity on what is required, please get in touch via email.

> Is there any way to get around this?

Unfortunately there is not - this is something your University has to fix.
